### PR TITLE
Fixed unknown placeholder issue

### PR DESCRIPTION
**Pull request by Shivakishore14**
Fixes the placeholder issue

### CODE REVIEW

File: chatbot.py [hunk 1]
```diff
@@ -105,18 +106,18 @@
 w3 =  tf.Variable(tf.random_normal([hidden_nodes2,output_classes]))
 b3 = tf.Variable(tf.random_normal([output_classes]))
 y = tf.nn.softmax(tf.matmul(l2_out,w3) + b3)
+y_max = tf.argmax(y,1)
```
Comment (Shivakishore14) [inline chatbot.py:109]: tf.argmax() gives the position of the maximum element present in `y`

File: chatbot.py [hunk 2]
```diff
@@ -105,18 +106,18 @@
...
 
 sess = tf.InteractiveSession()
 tf.global_variables_initializer().run()
 epochs = 2000
-
 for i in range(epochs):
     # print train_x[10],train_y[0]
-    print sess.run(train_step, feed_dict={ X:train_x, Y:train_y })
-    # print sess.run(y)
+    y_, _ = sess.run([y_max, train_step], feed_dict={ X:train_x, Y:train_y })
```
Comment (Shivakishore14) [inline chatbot.py:119]: Running both `train_step` and `y_max` at the same time.

File: chatbot.py [hunk 3]
```diff
@@ -105,18 +106,18 @@
...
 train_step = tf.train.GradientDescentOptimizer(0.005).minimize(cross_entropy)
 
 sess = tf.InteractiveSession()
 tf.global_variables_initializer().run()
 epochs = 2000
-
 for i in range(epochs):
     # print train_x[10],train_y[0]
-    print sess.run(train_step, feed_dict={ X:train_x, Y:train_y })
-    # print sess.run(y)
```
Comment (Shivakishore14) [inline chatbot.py:119]: this didn't work because it needs feed_dict={X:train_x},
because y depends on X for computations in graph

**Comment by vj-09** [comment]
Yet gives none value for train function.

**Comment by Shivakishore14** [comment]
it is supposed to give none for train operation,
as it changes the weights of the graph and doesn't return anything.

**Pull request merged**